README.md

```diff
@@ -1,16 +1,31 @@
-# sanitizers
-This project is the home for Sanitizers: AddressSanitizer, MemorySanitizer, ThreadSanitizer, LeakSanitizer, and more
-The actual code resides in the [LLVM](http://llvm.org) repository.
-Here we keep extended [documentation](../../wiki), [bugfixes](../../issues) and some helper code. 
+# sanitizers (Archived)
+
+**This project has been archived and is no longer actively maintained.**
+
+The Sanitizers project, which includes AddressSanitizer, MemorySanitizer, ThreadSanitizer, LeakSanitizer, and more, is now archived. The core code for these sanitizers resides within the [LLVM](http://llvm.org) repository. This repository will be retained for archival purposes, providing access to historical documentation, bugfixes, and helper code.
+
+**Important:** Please do not file new bug reports in this repository.
+
+**Where to Report Bugs:**
```

Comment:
It's unclear from this plus line 7 what will happen to existing bug reports, and what should users expect. Should they refile the bug reports at the locations below?

Reply:
For the existing bugs, I am going to either close them suggesting to file them at proper locations, or migrate the relevant bugs manually. This will be done on a case-by-case basis.